specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```
@@ -2589,6 +2589,51 @@
         "url": "https://docs.microsoft.com/rest/api/searchservice/Custom-analyzers-in-Azure-Search"
       }
     },
+    "NormalizerName": {
+      "type": "string",
+      "enum": [
+        "asciifolding",
+        "elision",
+        "lowercase",
+        "standard",
+        "uppercase"
+      ],
+      "x-ms-enum": {
+        "name": "NormalizerName",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "asciifolding",
+            "name": "AsciiFolding",
+            "description": "Converts alphabetic, numeric, and symbolic Unicode characters which are not in the first 127 ASCII characters (the \"Basic Latin\" Unicode block) into their ASCII equivalents, if such equivalents exist. See http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/miscellaneous/ASCIIFoldingFilter.html"
+          },
+          {
+            "value": "elision",
+            "name": "Elision",
+            "description": "Removes elisions. For example, \"l'avion\" (the plane) will be converted to \"avion\" (plane). See http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/util/ElisionFilter.html"
+          },
+          {
+            "value": "lowercase",
+            "name": "Lowercase",
+            "description": "Normalizes token text to lower case. See http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/core/LowerCaseFilter.htm"
```

"lowercase" and "uppercase" are both 1 word, as also indicated by the casing of the value and name. You should also use the single word equivalent in the description (here and elsewhere).
```suggestion
            "description": "Normalizes token text to lowercase. See http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/core/LowerCaseFilter.htm"
```

The link uses two words judging by their PascalCasing of LowerCase, but whether they do in the text of the document is impossible to tell: it 404s.

If you're trying to match what lucene does, I'd stick with their casing/nomenclature. 1- and 2-word variants are both acceptable. I'd just be consistent (at least throughout our swagger).

---

Thank you, I'll update the Swagger. :)

---

I think you have found a bug in our links. These were adapted from our existing `TokenFilter` section, and it appears that some of the links are broken.

---

The space is removed and link fixed. Thanks for catching that broken link.